public/quotes.json

```diff
@@ -3176,8 +3176,8 @@
       "tags": []
     },
     {
-      "quote": "",
-      "author": "",
+      "quote": "You are never too old to start over. Every day is a chance to make changes to create the life we want.",
+      "author": "Sandesh",
```

Comment:
```suggestion
      "author": "Karon Wadell",
```

Reply:
I have added suggestion, could you please make necessary changes to address this?

Thanks!